src/features/governance/components/Governance.tsx

```diff
@@ -1,20 +1,92 @@
+import { ProposalStage } from '@celo/contractkit/lib/wrappers/Governance';
+import { useEffect, useMemo } from 'react';
+import { ThemedIcon } from 'src/components/icons/ThemedIcon';
 import { Row } from 'src/components/list/row';
-import { Switcher } from 'src/components/switcher/Switcher';
-import { CenteredLayout } from 'src/layout/CenteredLayout';
-import { Mode } from 'src/types';
+import { StagePill } from 'src/features/governance/components/StagePill';
+import { useGovernance } from 'src/hooks/useGovernance';
 
 export const Governance = () => {
-  // const _ctx = useAccountContext();
+  const { loadDequeue, allProposals, error } = useGovernance();
+  useEffect(() => {
+    void loadDequeue();
+  }, [loadDequeue]);
+
+  const proposals = useMemo(
+    () =>
+      allProposals.filter((x) =>
+        [ProposalStage.Queued, ProposalStage.Referendum].includes(x.stage)
```

Comment:
nit : for small performance boost define this array outside the component and as a Set.